HW1/HomeWork3.py

```diff
@@ -1,12 +1,13 @@
-#Задача 6: Вы пользуетесь общественным транспортом? 
-# Вероятно, вы расплачивались за проезд и получали билет с номером. Счастливым билетом называют такой билет с шестизначным номером, 
-# где сумма первых трех цифр равна сумме последних трех. Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. 
+# Задача 6: Вы пользуетесь общественным транспортом?
+# Вероятно, вы расплачивались за проезд и получали билет с номером. Счастливым билетом называют такой билет с шестизначным номером,
+# где сумма первых трех цифр равна сумме последних трех. Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6.
 # Вам требуется написать программу, которая проверяет счастливость билета.
 
 number = int(input('Введите номер билета'))
```

Comment:
Если вы все равно используете введенное значение в качестве строки, давайте строкой его и оставим.
```suggestion
number = input('Введите номер билета')
```